README.md

```diff
@@ -97,10 +95,9 @@ Install this project with:
 
 ### Usage
 
-To execute the files of this project, run the following command:
+To execute the tests of this project, run the following command:
 ```sh
-  rails g controller UsersController index show
-  rails g controller PostsController index show
+  rspec --format documentation
 ```
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/229777263-6f9d812a-12de-4e8a-9f4b-7c56d37b5f1a.png)

**[OPTIONAL]**

- [ ] It would be great if you put rails server command in this section because this command is used to run your rails app so kindly add it and make your readme useful 👍